__tests__/app.test.js

```diff
@@ -394,4 +394,29 @@ describe("DELETE: /api/comments/:comment_id", () => {
     expect(msg).toBe("Bad request")
     })
   })
+})
+describe("GET /api/users", () => {
+  test("200: responds with an array of user objects", () => {
+    return request(app)
+    .get("/api/users")
+    .expect(200)
+    .then(({body: {users}})=> {
+      users.forEach((user)=> {
```

Comment:
Please check the length before iterating. Checking the length protects against you trying to iterate over an empty array (which, if that was the case, it would be skipped and so would all your assertions).